2023-03-15-dasharo-user-group-1.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Dasharo User Group #1 and Dasharo Developers vPub 0x6 (virtual event)"
+author: The Dasharo Team
+categories: announcements
+image: /attachment/posts/dasharo-event-1.png
+---
+
+_**Editor's note:** The following is an announcement from the Dasharo Team regarding an upcoming [virtual event](https://vpub.dasharo.com/e/1/dasharo-user-group-1)._
+
+[![Dasharo User Group (DUG) #1 and Dasharo Developers vPub 0x6 informational poster](/attachment/posts/dasharo-event-1.png)](https://vpub.dasharo.com/e/1/dasharo-user-group-1)
+
+## Dasharo User Group (DUG) #1 and Dasharo Developers vPub 0x6
+
+### When? 16 March 2023 from 5 PM UTC to last haker standing
+
+**The Dasharo User Group (DUG)** is an important forum for users of Dasharo to come together, share their knowledge, and stay informed about the latest developments in the Dasharo ecosystem. The DUG is a platform for users to connect and learn about new features and updates that are coming to Dasharo. The first DUG event will take place in early March and will include a variety of discussions on different topics related to Dasharo. The event will be a great opportunity for Dasharo users to meet other users, learn new things, and share their knowledge and experience with others.
+
+**The Dasharo Developers vPub** is a virtual event with goal for gathering developers and enthusiasts in the technology industry to engage in discussions and conversations related to open-source firmware and other related topics. The vPub is designed to be a less structured, more relaxed environment where participants can share knowledge and ideas, connect with others in the industry, and learn about the latest developments in the field. The conversations can range from technical challenges faced by developers, off-topic discussions, to new features or improvements. These events have brought together experts and enthusiasts from all over the world, providing valuable opportunities for learning, collaboration, and networking. Everything in casual atmosphere with beverage of your choice in your hand.
+
+[Read the full announcement for more information.](https://vpub.dasharo.com/e/1/dasharo-user-group-1)
```

Comment:
@andrewdavidwong does it make sense to mention MSI certifcation or internally link to it?